auth0-variables.js

```diff
@@ -0,0 +1,3 @@
+var AUTH0_CLIENT_ID='KvwsETaUxuXVjW2cmz2LbdqXQBdYs6wH'; 
```

Comment:
no more `var`s in ES6. use const. also, this file doesn't seem to be used anywhere, and consts.js seems to have the same data.

looks like you don't need this file?